README.md

```diff
@@ -1,3 +1,17 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
Not sure where this is coming from? Could we disable this?

Reply:
It might be from the crowdin thing or documentation generator (certainly not the `./cli.js generate`).

Reply:
Ooh, do we need to keep that?

Reply:
I hope we can remove it 🙏